WinnersProx/AutoMart
# Bg backend fixes 166663340

#### What does this PR do?
This pull request implements all working API endpoints for the project
#### Description of Task to be completed?
For this, I've tried to correct some errors in order to present a work that is free of errors and bugs
#### How should this be manually tested?
To test this manually this repo should be cloned and then run ```npm install```  and   ```npm start```
#### Any background context you want to provide?
Actually none
#### What are the relevant pivotal tracker stories?
it is ```fix api endpoints bugs for delivery with an id #166663340 ```
